roles/zabbix_server/vars/RedHat.yml

```diff
@@ -9,6 +9,8 @@ __epel_repo:
   - epel
 
 zabbix_valid_server_versions:
+  "10":
+    - 7.4
```

Comment:
Looks like there are packages for 7.0? https://repo.zabbix.com/zabbix/7.0/rhel/10/x86_64/

Reply:
You know I thought there would be but it didn't show up on their downloads page, and I didn't think about looking in the repo.  Will update after I do Debian 13 (which is being a pain)